docs/components/overrides/Editor/Editor.js

```diff
@@ -7,12 +7,12 @@ import 'codemirror/mode/jsx/jsx'
 // We’re explicitly specifying Webpack loaders here so we could skip specifying them in Webpack configuration.
 // That way we could avoid clashes between our loaders and user loaders.
 // eslint-disable-next-line import/no-unresolved, import/no-webpack-loader-syntax
-require('!!../../../node_modules/react-styleguidist/loaders/style-loader!../../../node_modules/react-styleguidist/loaders/css-loader!../../../node_modules/codemirror/lib/codemirror.css')
+require('!!../../../../node_modules/react-styleguidist/loaders/style-loader!../../../../node_modules/react-styleguidist/loaders/css-loader!../../../../node_modules/codemirror/lib/codemirror.css')
 
 // loading our own (TDS) custom AAA colour accessible ttcn.css theme instead of loading the ttcn theme from the
 // codemirror package mapped to the rsg-codemirror-theme.css webpack alias
 // eslint-disable-next-line import/no-unresolved, import/no-webpack-loader-syntax
```

Comment:
Please remove

Reply:
I tried to remove this...... but then our editors didn't format correctly. Seems that we still need this.